client/package.json

```diff
@@ -3,66 +3,61 @@
-    "lint:fix": "ng lint --fix && yarn lint:sass",
-    "lint:sass": "sass-lint --verbose --no-exit",
-    "format:fix:staged": "pretty-quick --staged --verbose --pattern \"**/*.{js,json,md,ts}\""
+    "lint": "ng lint",
+    "lint:fix": "ng lint --fix"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~11.0.0",
-    "@angular/cdk": "^10.2.7",
-    "@angular/common": "~11.0.0",
-    "@angular/compiler": "~11.0.0",
-    "@angular/core": "~11.0.0",
-    "@angular/forms": "~11.0.0",
-    "@angular/material": "^10.2.7",
-    "@angular/platform-browser": "~11.0.0",
-    "@angular/platform-browser-dynamic": "~11.0.0",
-    "@angular/router": "~11.0.0",
+    "@angular/animations": "~12.1.2",
+    "@angular/cdk": "^12.1.2",
+    "@angular/common": "~12.1.2",
+    "@angular/compiler": "~12.1.2",
+    "@angular/core": "~12.1.2",
+    "@angular/forms": "~12.1.2",
+    "@angular/material": "^12.1.2",
+    "@angular/platform-browser": "~12.1.2",
+    "@angular/platform-browser-dynamic": "~12.1.2",
+    "@angular/router": "~12.1.2",
     "@briebug/jest-schematic": "^2.1.1",
-    "@schuchard/prettier": "^3.1.0",
+    "@schuchard/prettier": "^5.0.0",
     "rxjs": "~6.5.4",
     "tslib": "^2.0.0",
-    "zone.js": "~0.10.2"
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1100.0",
-    "@angular-eslint/builder": "0.6.0-beta.0",
-    "@angular-eslint/eslint-plugin": "0.6.0-beta.0",
-    "@angular-eslint/eslint-plugin-template": "0.6.0-beta.0",
-    "@angular-eslint/schematics": "^0.6.0-beta.0",
-    "@angular-eslint/template-parser": "0.6.0-beta.0",
-    "@angular/cli": "~11.0.0",
-    "@angular/compiler-cli": "~11.0.0",
-    "@angular/language-service": "~11.0.0",
-    "@ngneat/spectator": "^5.3.1",
+    "@angular-devkit/build-angular": "~12.1.2",
+    "@angular-eslint/builder": "^12.0.0",
+    "@angular-eslint/eslint-plugin": "^12.0.0",
+    "@angular-eslint/eslint-plugin-template": "^12.0.0",
+    "@angular-eslint/schematics": "^12.3.1",
+    "@angular-eslint/template-parser": "^12.0.0",
+    "@angular/cli": "~12.1.2",
+    "@angular/compiler-cli": "~12.1.2",
+    "@angular/language-service": "~12.1.2",
+    "@ngneat/spectator": "^8.0.1",
     "@types/jest": "25.1.4",
-    "@types/node": "^12.11.1",
-    "@typescript-eslint/eslint-plugin": "4.3.0",
-    "@typescript-eslint/parser": "4.3.0",
+    "@types/node": "^16.3.3",
+    "@typescript-eslint/eslint-plugin": "4.28.2",
+    "@typescript-eslint/parser": "4.28.2",
     "bootstrap": "^4.4.1",
-    "eslint": "^7.6.0",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-plugin-prettier": "^3.1.4",
+    "eslint": "^7.31.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^3.4.0",
     "husky": "4.2.3",
     "jest": "^25.2.3",
     "jest-preset-angular": "8.1.2",
     "lint-staged": "10.0.9",
-    "prettier": "2.1.2",
-    "pretty-quick": "^2.0.1",
+    "prettier": "^2.3.2",
```

Comment:
Since the tasks for them are gone, do we still need `prettier` and `pretty-quick` here or is the linting rule enough?

Reply:
Right, the linting rule is enough, removed the `prettier` stuff